tools/sync-translations.js

```diff
@@ -0,0 +1,98 @@
+/*!
+ * Copyright 2013 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://www.osedu.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var _ = require('underscore');
+var config = require(__dirname + '/../config');
+var Crowdin = require('crowdin');
+var fs = require('fs');
+var temp = require('temp');
+
+// Automatically track and cleanup files at exit
+temp.track();
+
+var crowdin = new Crowdin(config.crowdin);
```

Comment:
I would prefer not using a config and just passing in the parameters through the command line, which is part of the alignment between our crowdin synchs. It's probably also not necessary to have certain things like the project id in the config file (as we will only ever have 1 crowdin project)